src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/ObservationMapperTest.java

```diff
@@ -102,17 +140,19 @@ void testMapToObservationsLaboratoryInvestigationMissingCode() {
     }
 
     @Test
-    void testMapToTestGroupsAndResultsLaboratoryInvestigationResult() {
-        final Message message = new Message(List.of(
+    void testMapLaboratoryInvestigationResult() {
+        final var message = new Message(List.of(
             "S02+02", // ServiceReportDetails
             "S06+06", // InvestigationSubject
             "GIS+N", // LabResult
             "INV+MQ+c:911::d", // LaboratoryInvestigation
+            "RFF+ASL:1", // Reference
 
```

Comment:
```suggestion
```

Reply:
Or is there a purpose?

Reply:
They're not optional anyway and now we're actually consuming them - expecting them to be there.